tests/testTriggers.go

```diff
@@ -5742,6 +5744,107 @@ func TriggerAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *EventRecor
 	updateMetrics(*event)
 }
 
+// TriggerAsyncDRVolumeOnly triggers Async DR
+func TriggerAsyncDRVolumeOnly(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	dash.Infof("Volume Only Async DR triggered at: %v", time.Now())
```

Comment:
add startlongevity and endlongevity calls to your test

Reply:
Done.

Reply:
start and end longevity tests are not added?

Reply:
Added, Thanks!